### PR TITLE
chore: add storybook

### PR DESCRIPTION
**Pull request by stipsan**
Adds a basic Storybook for `@sanity/visual-editing` with `@sanity/demo` setup to get us started.

You start it locally by running `pnpm storybook:visual-editing`. [It's especially nice if you install the VSCode extension for it](https://marketplace.visualstudio.com/items?itemName=joshbolduc.story-explorer):

image

I'll setup a Vercel deploy for it after this merges 🙌

### CODE REVIEW

File: packages/visual-editing/package.json [hunk 1]
```diff
@@ -164,29 +168,44 @@
...
+    "@storybook/test": "^8.2.9",
     "@sveltejs/adapter-auto": "^3.2.3",
     "@sveltejs/kit": "^2.5.21",
     "@sveltejs/package": "^2.3.3",
+    "@tailwindcss/typography": "^0.5.14",
     "@types/react": "^18.3.3",
     "@types/react-dom": "^18.3.0",
     "@typescript-eslint/eslint-plugin": "^7.18.0",
     "@typescript-eslint/parser": "^7.18.0",
+    "add": "^2.0.6",
```
Comment (rdunk) [inline packages/visual-editing/package.json:190]: Was this added intentionally?

File: packages/visual-editing/package.json [hunk 2]
```diff
@@ -164,29 +168,44 @@
...
     "eslint": "^8.57.0",
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-prettier": "^5.2.1",
     "eslint-plugin-react": "^7.35.0",
     "eslint-plugin-react-hooks": "^4.6.2",
     "eslint-plugin-simple-import-sort": "^12.1.1",
+    "eslint-plugin-storybook": "^0.8.0",
     "happy-dom": "^14.12.3",
     "ls-engines": "^0.9.3",
+    "npx": "^10.2.2",
```
Comment (rdunk) [inline packages/visual-editing/package.json:200]: Was this added intentionally?

Comment (stipsan) [reply packages/visual-editing/package.json:200]: lol, no! Good catch!!!

File: packages/visual-editing/package.json (hunk 1 above)
Comment (stipsan) [reply packages/visual-editing/package.json:190]: No! I think I ran something like `pnpm add npx add` by accident 🤦

**Review by rdunk — APPROVED** [review]

**Pull request merged**